.jenkinsci/debug-build.groovy

```diff
@@ -5,12 +5,16 @@ def doDebugBuild(coverageEnabled=false) {
   def manifest = load ".jenkinsci/docker-manifest.groovy"
   def pCommit = load ".jenkinsci/previous-commit.groovy"
   def parallelism = params.PARALLELISM
+  def sanitizeEnabled = params.sanitize
   def platform = sh(script: 'uname -m', returnStdout: true).trim()
   def previousCommit = pCommit.previousCommitOrCurrent()
   // params are always null unless job is started
   // this is the case for the FIRST build only.
   // So just set this to same value as default.
   // This is a known bug. See https://issues.jenkins-ci.org/browse/JENKINS-41929
+  if (sanitizeEnabled == null){
```

Comment:
The value defaults to `true`. Why do you ever need to check for null?

Reply:
see comment in line 11, params can be `null`
> // params are always null unless job is started
  // this is the case for the FIRST build only.
  // So just set this to same value as default.
  // This is a known bug. See https://issues.jenkins-ci.org/browse/JENKINS-41929

Reply:
sanitizeEnabled can be set as `false` and  suggested change will set it to true:

`sanitizeEnabled == null ? true ` may be better solutions, but I keep old one and leave this for future refactoring, because we may want change all simple `if/else` block